File: Expiring Bread.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int n,m,k;
        scanf("%d%d%d",&n,&m,&k);
        if(k*m>=n)
        printf("YES\n");
        else
        printf("NO\n");
    }
}

File: Expiring Bread.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int n=sc.nextInt();
            int m=sc.nextInt();
            int k=sc.nextInt();
            if(m*k>=n)
            System.out.println("YES");
            else
            System.out.println("NO");
        }
    }
}